webview/src/plots/components/comparisonTable/ComparisonTableRow.tsx

```diff
@@ -128,21 +134,47 @@ export const ComparisonTableRow: React.FC<ComparisonTableRowProps> = ({
             className={cx({ [styles.pinnedColumnCell]: pinnedColumn })}
             colSpan={pinnedColumn ? 1 : nbColumns}
           >
-            <div className={styles.rowPath}>
-              <button
-                className={styles.rowToggler}
-                onClick={toggleIsShownState}
-              >
-                <Icon icon={isShown ? ChevronDown : ChevronRight} />
-                <Tooltip
-                  content={path}
-                  placement="bottom-start"
-                  delay={NORMAL_TOOLTIP_DELAY}
+            <div className={styles.rowHeader} data-testid="row-header">
+              <div className={styles.rowHeaderPath}>
+                <button
+                  className={styles.rowToggler}
+                  onClick={toggleIsShownState}
                 >
-                  <span className={styles.pathText}>{path}</span>
-                </Tooltip>
-              </button>
-              <CopyButton value={path} className={styles.copyButton} />
+                  <Icon icon={isShown ? ChevronDown : ChevronRight} />
+                  <Tooltip
+                    content={path}
+                    placement="bottom-start"
+                    delay={NORMAL_TOOLTIP_DELAY}
+                  >
+                    <span className={styles.pathText}>{path}</span>
+                  </Tooltip>
+                </button>
+                <CopyButton value={path} className={styles.copyButton} />
+              </div>
+              {boundingBoxLabelsArr.length > 0 && (
+                <div className={styles.boundingBoxLabels}>
+                  <p className={styles.boundingBoxLabelsTitle}>Labels</p>
```

Comment:
I just realized I've been using the term "labels" instead of classes which makes more sense 🤦‍♀️. I'll need to update the term in variables and types as well as the text.

Reply:
I think either is acceptable @julieg18. I also used "labels" in https://github.com/iterative/dvclive/issues/766 and have seen both used in other products.

Reply:
Oh, good to know! I've already renamed everything to classes so I suppose we'll just keep it as classes 😄.